grails-app/conf/Config.groovy

```diff
@@ -1135,6 +1135,7 @@ openboxes.receiving.receivingLocation.prefix = Constants.DEFAULT_RECEIVING_LOCAT
 
 // Pagination
 openboxes.api.pagination.enabled = true
+openboxes.api.pagination.minimumBatchSize = 50
```

Comment:
Set this to 10 by default. And get rid of the initial value of 30. I don't care if the user sees "Loading..." under the first 10 rows, I just want them to see those rows as quickly as possible.